src/_guides/language/language-tour.md

```diff
@@ -160,22 +166,23 @@ The following table lists the words that the Dart language treats specially.
 | deferred{{bii}}   | implements{{bii}}
 {:.table .table-striped .nowrap}
 
-<sup>1</sup> Words with the superscript **1**
-are **built-in identifiers**. Avoid using
-built-in identifiers as identifiers.
-A compile-time error happens if you try to
-use a built-in identifier for a class or type name.
-
-<sup>2</sup> Words with the superscript **2**
-are newer, limited reserved words related to **asynchrony** support
-added after Dart's 1.0 release.
-You can't use `async`, `await`, or `yield` as
-an identifier in any function body marked with `async`, `async*`, or `sync*`.
-For more information, see
-[Asynchrony support](#asynchrony-support).
+When necessary, the words marked with superscripts can sometimes be used as
+identifiers:
+
+* Words with the superscript **1** are **built-in identifiers**.
+  To simplify the task of porting JavaScript code to Dart,
+  built-in identifiers are valid in most places,
+  but they can't be used as class or type names.
```

Comment:
Maybe say, "they can't, e.g., be used as ..." because there is one more restriction (can't be used as an import prefix), or add that to the list.